composer.json

```diff
@@ -15,13 +15,14 @@
     "api-clients/hydrator": "dev-master",
     "api-clients/service": "dev-master",
     "api-clients/transport": "dev-master",
-    "league/event": "^2.1",
-    "php-di/php-di": "^5.4"
```

Comment:
Not 100% sure if we should do this. The foundation providers a container for the application/client, and patching the leakage should hide it from the application. If we want to give applications access to adding definitions we could use the same kind of construction as we do with the command bus, some extra configuration in `composer.json`: https://github.com/php-api-clients/github/blob/master/composer.json#L38-L45

(No need to do that in this PR though :smile: .)